bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -147,9 +147,7 @@
 
     private final AtomicBoolean rmRegistered = new AtomicBoolean(false);
```

Comment:
I think `rmRegistered` and `forceReadOnly` should be part of state manager as well.